lib/plugins/inspec-parallel/lib/inspec-parallel/command.rb

```diff
@@ -77,6 +91,7 @@ def read_options_file
         end
         content.each.with_index(1) do |str, index|
           data_hash = { line_no: index }
+          str = "<% pid = 'CHILD_PID' %>" + str if str.include? "pid"
```

Comment:
You must actually evaluate the variable pid with the value CHILD_PID. You cannot predict what the string is going to contain.